SPECS/libsolv/libsolv.spec

```
@@ -46,7 +46,8 @@ Requires:       xz
     -DENABLE_RPMDB=ON                   \
     -DENABLE_RPMDB_BYRPMHEADER=ON       \
     -DENABLE_RPMDB_LIBRPM=ON            \
-    -DENABLE_RPMMD=ON
+    -DENABLE_RPMMD=ON                   \
+	-DENABLE_COMPS=ON
```

Nit: lonely tab in a sea of spaces

```suggestion
    -DENABLE_COMPS=ON
```